core/extraction/round_1.py

```diff
@@ -0,0 +1,208 @@
+import pandas as pd
+
+
+def ingest_round_1_data(df_map: pd.DataFrame) -> dict[pd.DataFrame]:
+    """
+    Extract data from Round 1 Reporting Template into column headed Pandas DataFrames.
+
+    :param df_ingest: DataFrame of parsed Excel data.
+    :return: Dictionary of extracted "tables" as DataFrames.
+    """
+    df_dictionary = dict()
+
+    df_dictionary["programme_outcomes"] = extract_programme_outcomes(df_map["programme_outcomes"])
+    df_dictionary["programme_risks"] = extract_programme_risks(df_map["programme_risks"])
+    df_dictionary["programme_summary"] = extract_programme_summary(df_map["programme_summary"])
+    df_dictionary["project_funding_comments"] = extract_project_funding_comments(df_map["project_funding_comments"])
+    df_dictionary["project_funding_profiles"] = extract_project_funding_profiles(
+        df_map["project_funding_profiles_MASTER"]
+    )
+    df_dictionary["project_location"] = extract_location(df_map["project_location"])
+    df_dictionary["project_progress"] = extract_project_progress(df_map["project_progress"])
+    df_dictionary["project_risks"] = extract_project_risks(df_map["project_risks"])
+    df_dictionary["postcode_region"] = extract_postcode_region(df_map["lookup_postcode_region"])
+    df_dictionary["programme_project"] = extract_programme_project(df_map["lookup_programme_project"])
+    df_dictionary["td_fundingcdel_rdel_accelerated"] = extract_accelerated(df_map["td_fundingcdel_rdel_accelerated"])
+    df_dictionary["programme_management"] = extract_programme_management(df_map["td_funding_programme_management"])
+
+    return df_dictionary
+
+
+def extract_programme_outcomes(df_programme: pd.DataFrame) -> pd.DataFrame:
+    """
+    Extract programme outcomes information from a DataFrame.
+
+    Input dataframe is parsed from Excel spreadsheet: "Round 1 Reporting - TF_aggregated_data_DD.MM.YYYY.xlsx".
+    Specifically Project work sheet, parsed as dataframe.
+
+    :param df: Input DataFrame containing data.
+    :return: Extracted DataFrame containing programme data.
+    """
+
+    df_programme = df_programme.reset_index(drop=True)
```

Comment:
I don't think this actually does anything as we haven't altered the dataframe at all (I just checked on debugger and the indexes look ok without resetting them)